app/controllers/vendor_api/applications_controller.rb

```diff
@@ -25,12 +35,7 @@ def application_choices_visible_to_provider
         .call(
           providers: [current_provider],
           vendor_api: true,
-          includes: [
```

Comment:
I think this might need to be rebase as the `ApplicationDataConcerns` has been merged